### PR TITLE
fix: Add hackathon resources and refactor to use enums

### PR DESCRIPTION
**Pull request by jeremytchang**
- Add JOIN Yt videos to hackathon resources
- Updated resource tags and type to use enums for better maintenance
- Updated resource buttonitem rendering to be based off tags enum

### CODE REVIEW

File: packages/hackathon/src/scenes/ResourceScene/resource_data.ts [hunk 1]
```diff
@@ -24,400 +24,455 @@
...
 
+// Enum serves as display strings too
+enum ResourceType {
+  Resource = 'Resource',
+  Tutorial = 'Tutorial',
+  Example = 'Example',
+}
+
+// Enum serves as display strings too
+export enum ResourceTag {
```
Comment (jkaster) [inline packages/hackathon/src/scenes/ResourceScene/resource_data.ts:35]: Handier

File: packages/hackathon/src/scenes/ResourceScene/resource_data.ts [hunk 2]
```diff
@@ -24,400 +24,455 @@
 
  */
 
+// Enum serves as display strings too
+enum ResourceType {
```
Comment (jkaster) [inline packages/hackathon/src/scenes/ResourceScene/resource_data.ts:28]: Handy

File: packages/hackathon/src/scenes/ResourceScene/resource_data.ts [hunk 3]
```diff
@@ -24,400 +24,455 @@
...
+  CustomViz = 'Custom Viz',
+  Other = 'Other',
+}
+
 export interface Resource {
   content: string
-  type: string
-  tag: string
+  type: ResourceType
+  tag: ResourceTag
```
Comment (jkaster) [inline packages/hackathon/src/scenes/ResourceScene/resource_data.ts:51]: Handiest!

**Review by jkaster — APPROVED** [review]
Nice improvement

File: packages/hackathon/src/scenes/ResourceScene/resource_data.ts (hunk 2 above)
Comment (jeremytchang) [reply packages/hackathon/src/scenes/ResourceScene/resource_data.ts:28]: ✋

File: packages/hackathon/src/scenes/ResourceScene/resource_data.ts (hunk 1 above)
Comment (jeremytchang) [reply packages/hackathon/src/scenes/ResourceScene/resource_data.ts:35]: ✋

File: packages/hackathon/src/scenes/ResourceScene/resource_data.ts (hunk 3 above)
Comment (jeremytchang) [reply packages/hackathon/src/scenes/ResourceScene/resource_data.ts:51]: ✋

**Pull request merged**